--- Python-2.7.3/configure.orig 2013-07-10 09:47:19.730106467 +0200
+++ Python-2.7.3/configure      2013-07-10 09:47:44.214132511 +0200
@@ -5207,16 +5207,9 @@
 fi
 
 
-if test $HAS_HG = found
-then
-    HGVERSION="hg id -i \$(srcdir)"
-    HGTAG="hg id -t \$(srcdir)"
-    HGBRANCH="hg id -b \$(srcdir)"
-else
     HGVERSION=""
     HGTAG=""
     HGBRANCH=""
-fi
 
 case $MACHDEP in
 bsdos*|hp*|HP*)
